elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -6,19 +6,49 @@
 
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
-    pl.check_attribs(element, required_attribs=[], optional_attribs=['digits'])
+    required_attribs = []
+    optional_attribs = ['digits', 'default-tab', 'show-matlab', 'show-mathematica', 'show-python']
+    pl.check_attribs(element, required_attribs, optional_attribs)
 
 
 def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     digits = pl.get_integer_attrib(element, 'digits', 2)
+    display_matlab_tab = pl.get_boolean_attrib(element, 'show-matlab', True)
+    display_mathematica_tab = pl.get_boolean_attrib(element, 'show-mathematica', True)
+    display_python_tab = pl.get_boolean_attrib(element, 'show-python', True)
+    default_tab = pl.get_string_attrib(element, 'default-tab', 'matlab')
+
+    # Default active tab
+    if default_tab == 'matlab':
+        active_tab_matlab = True
```

Comment:
How about setting these before the `if` to be `False` and then inside the appropriate `if` body set the specific value to `True`? 

e.g. 
```py

# Initialize values to false
active_tab_matlab = False
active_tab_mathematica = False
active_tab_python = False

# Set starting tab
if default_tab == 'matlab':
    active_tab_matlab = True
elif default_tab == 'mathematica': 
    active_tab_mathematica = True
... 
```

Reply:
This is way more elegant and less repetitive. I also changed up my default tab handling. 

Do I need to throw an error if someone tries to set the default tab to one that they have hidden? Right now the code just handles it and basically ignores the impossible `default-tab` setting.

Reply:
I think it’s fine to just silently work the best we can in this case.